### PR TITLE
mgr/dashboard: Add hints to resolve unit test failures

### PR DESCRIPTION
**Pull request by Devp00l**
Add hints to resolve unit test failures

### CODE REVIEW

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 1]
```diff
@@ -85,6 +85,12 @@ Create ``unit-test-configuration.ts`` file based on
 Run ``npm run test`` to execute the unit tests via `Jest
 <https://facebook.github.io/jest/>`_.
 
+If you get errors on all tests, it could be the reason that `Jest
+<https://facebook.github.io/jest/>`_ or something else were updated.
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/HACKING.rst]: s/were/was

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 2]
```diff
@@ -85,6 +85,12 @@ Create ``unit-test-configuration.ts`` file based on
...
+There are a view ways how you can try to resolve this:
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/HACKING.rst]: s/view/few

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 3]
```diff
@@ -85,6 +85,12 @@ Create ``unit-test-configuration.ts`` file based on
...
+- Remove all modules and run ``npm install`` again in order to reinstall them
```
Comment (votdev) [inline src/pybind/mgr/dashboard/HACKING.rst]: Maybe you should add the info how to remove the modules, e.g. ``rm -rf dist/ node_modules/ package-lock.json``.

Comment (tspmelo) [reply src/pybind/mgr/dashboard/HACKING.rst]: I would not recommend removing `package-lock.json`.

**Comment by Devp00l** [comment]
Addressed all comments

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 4]
```diff
@@ -85,6 +86,14 @@ Create ``unit-test-configuration.ts`` file based on
 Run ``npm run test`` to execute the unit tests via `Jest
 <https://facebook.github.io/jest/>`_.
 
+If you get errors on all tests, it could be the reason that `Jest
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/HACKING.rst]: s/it could be the reason that/it could be because/

**Comment by Devp00l** [comment]
Addressed all comments

**Pull request merged**